agent/doc/swdesign/README.md

```diff
@@ -256,203 +317,240 @@ Needs:
 
 Tags:
-- AgentManager
-- ParameterStorage
+- RuntimeManager
 
 Needs:
 - impl
 - utest
 
-#### Handling the initial UpdateWorkload after Agent start (starting the RuntimeAdapters)
+#### Handling the initial UpdateWorkload after Agent start
 
 The following diagram and the subsequent requirements show the steps the Ankaios Agent takes when receiving the first UpdateWorkload command sent by Server. The first UpdateWorkload contains the complete initial list of workloads the Agent shall manage.
 
+
 ![Handling initial UpdateWorkload](plantuml/seq_update_workload_initial.svg)
 
-##### Agent Manager starts Runtime Adapters with initial list of workloads
-`swdd~agent-starts-runtimes-adapters-with-initial-workloads~1`
+##### RuntimeManager initial list of workloads handles existing workloads
+`swdd~agent-initial-list-existing-workloads~1`
 
 Status: approved
 
-After receiving the complete list of added workloads from the Ankaios Sever at the initial connection establishment, the Ankaios Agent shall start each RuntimeAdapter with the list of Workloads associated with this adapter.
+After receiving the complete list of added workloads from the Ankaios Sever at the initial connection establishment, the RuntimeManager shall handle existing workloads.
 
 Comment:
-In case the Agent was already running, the Runtime Adapter can take care of Workloads that were started in an earlier execution.
+In case the Agent was already running, the Runtime Adapter can take care of Workloads that were started in an earlier execution. Some of these workloads can be reused, some have to be updated and some stopped.
 
 Tags: 
-- AgentManager
+- RuntimeManager
 
 Needs:
 - impl
 - utest
 
-##### Runtime Adapter at startup finds existing Workloads
-`swdd~agent-adapter-start-finds-existing-workloads~1`
+##### RuntimeManager handles existing workloads by building list
+`swdd~agent-existing-workloads-finds-list~1`
 
 Status: approved
 
-The RuntimeAdapter shall build a list of existing Workloads started during the same machine runtime window by a previous execution of an Ankaios Agent with the same name as the currently running Agent.
+When handling existing workloads, the RuntimeManager shall call each RuntimeFacade to request a list of existing workloads started during the same machine runtime window by a previous execution of an Ankaios Agent with the same name as the currently running Agent.
 
 Comment:
-A 'machine runtime window' is the time between the start and shutdown of the machine.
+A 'machine runtime window' is the time between the start and shutdown of the machine. Finding existing workloads needs to be done fore stating new workloads in order to avoid conflicts. If this call fails, the agent currently ignores the failure assumes that no workloads are running. It must be confirmed that this behavior is correct.
```

Comment:
```suggestion
A 'machine runtime window' is the time between the start and shutdown of the machine. Finding existing workloads needs to be done before stating new workloads in order to avoid conflicts. If this call fails, the agent currently ignores the failure assumes that no workloads are running. It must be confirmed that this behavior is correct.
```